docs/pages/tutorial/eas/introduction.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: 'EAS Tutorial: Introduction'
+sidebar_title: Introduction
+description: An introduction to the tutorial for building apps for Android and iOS using Expo Application Services (EAS) that covers the Build, Update, and Submit workflows.
+---
+
+import Highlight from '~/components/plugins/Highlight';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { BookOpen02Icon } from '@expo/styleguide-icons';
+
+We're about to embark on a journey for building our mobile apps on the cloud. In this tutorial, we'll take our app built in the previous tutorial, create a development build and a preview version, and deploy it to the app stores.
+
+## What will I learn in this tutorial
+
+The objective of this tutorial is to get familiar with [Expo Application Services (EAS)](https://expo.dev/eas) such as Build, Submit, and Update. The following topics are covered:
+
+- Use EAS Build to create and install a development build, then run it on a device, emulator, or simulator.
+- Discover the benefits of using a development build.
+- Implement workflows for sharing development builds with a team or external stakeholders.
+- Efficiently handle different app versions.
+- Simultaneously install different app variants, like development and preview, on one device.
+- Utilize EAS Update to create and deploy updates swiftly during the development phase.
+- Automate build processes by integrating with a GitHub repository.
+
+These topics will give us a foundation to learn the fundamentals of using different EAS services.
+
+## How to use this tutorial
+
+This tutorial emphasizes learning by doing. It is self-paced and can take up to two to three hours to complete. To follow along, you can use your own project if you have it already, or you can choose one of the following options:
+
+- Use the Sticker Smash app from the previous tutorial. If you haven't followed the previous tutorial, download the complete source code from [this GitHub repository](https://github.com/amandeepmittal/sticker-smash-expo-tutorial-app) and set the project on your local machine. **We're going to use this as our example app**.
+- Create a new project with [`npx create-expo-app`](/get-started/create-a-project/).
+- Create and use a project generated by React Native CLI (also known as bare React Native project). It will contain the native directories (**android** and **ios**). You'll have to install the `expo` package (either [automatically](/bare/installing-expo-modules/) or [manually](/bare/installing-expo-modules/#manual-installation)) in the project before you proceed.
```

Comment:
Do you mean changing the title of the section to Prerequisites? Or create a sub section here for it?

Reply:
I think I originally meant having a section called "Prerequisites", but looking back again, just renaming the section would work as well, because it tells you, in order to do this tutorial, you need a React Native project: either use an existing project, create one or use Sticker Smash